README.md

```diff
@@ -27,10 +29,10 @@ Full video can be found at https://www.youtube.com/watch?v=sW5ZIcfX7w8
 
 ## Technologies used
 
-1. DHT - this is our strongest side that open way to build fully distributed and secured private networking
-2. STUN
-3. AES-256 - traffic encription
-4. uTP node communication
+1. **DHT** - distributed node discovery (based on seed IP address list)
+2. **STUN** - NAT traversal
```

Comment:
As I can see STUN is used only for NAT behavior discovery, not for traversal.
Also UPnP is used to forward ports, but it is still not traversal, nor STUN :)